pandas/core/index.py

```diff
@@ -1494,7 +1512,10 @@ def get_indexer(self, target, method=None, limit=None):
 
         Returns
         -------
-        indexer : ndarray
+        indexer : ndarray of int
+            Integers from 0 to n - 1 indicating that the index at these
+            positions matches the corresponding target values. Missing values
+            in the target are marked by -1.
         """
         method = self._get_method(method)
```

Comment:
right, this should raise ``ValueError`` if its not valid (I made a comment below about this). So method should be good after here.